src/components/spark/spark.component.html

```diff
@@ -11,7 +11,7 @@
         </div>
 
         <div class="ux-spark ux-inline" [style.height.px]="barHeight" [style.backgroundColor]="trackColor" [tooltip]="tooltip">
-            <div class="ux-spark-bar" [style.width]="(value < 100 ? value : 100) + '%'" [style.backgroundColor]="barColor"></div>
+            <div class="ux-spark-bar" *ngFor="let line of value; let idx = index;" [style.width.%]="line" [style.backgroundColor]="barColor[idx]"></div>
```

Comment:
Can we reference values instead of value here, as every time value is checked it has to call a function which is more expensive than just looking up an array